node-require/calculate.js

```diff
@@ -0,0 +1,42 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+const num1 = Number(process.argv[2]);
+const operator = process.argv[3];
+const num2 = Number(process.argv[4]);
+
+const operationsObj = {
+  plus: add,
+  minus: subtract,
+  times: multiply,
+  over: divide
+};
+
+let result;
+let errorFound = false;
+
+for (const key in operationsObj) { // check operations object and evaluate result if operation exists
+  if (operator === key) {
+    const desiredOperation = operationsObj[key];
+    result = desiredOperation(num1, num2);
+    if (result % 1 !== 0) { // catch non-whole numbers -- round to 2 decimals
+      result = result.toFixed(2);
+    }
+  }
+}
```

Comment:
You don't need a loop to figure out if a specific key exists in an object. You can do:
```JAVASCRIPT
if (operationsObj[operator]) {
  const desiredOperation = operationsObj[operator];
  result = desiredOperation(num1, num2);
  if (result % 1 !== 0) { // catch non-whole numbers -- round to 2 decimals
    result = result.toFixed(2);
  }
}
```

If the "operator" doesn't exist as a property in the object it will be `undefined` which is falsey. If it does exist it will be the function which is truthy.